.gitignore

```diff
@@ -0,0 +1,79 @@
+# Project specific
+oca.cfg
```

Comment:
What is this oca.cfg?

Reply:
No idea, but it was added in https://github.com/OCA/maintainer-quality-tools/pull/518. Should I remove it?

cc @MiquelRForgeFlow

Reply:
I'll remove this one, as I see no use for it. Ping me otherwise please.